include/seqan3/core/add_enum_bitwise_operators.hpp

```diff
@@ -18,6 +18,78 @@
 
 namespace seqan3
 {
+/*!\interface seqan3::enum_bitwise_operators
+ * \brief You can expect these functions on all types that implement seqan3::add_enum_bitwise_operators.
+ */
+/*!\name Requirements for seqan3::enum_bitwise_operators
```

Comment:
You can leave the comment open, however it's not wrong what you did... just pointing it out.
```suggestion
 *
 * \name Requirements for seqan3::enum_bitwise_operators
```

Reply:
I got some warning messages then, don't know why, but that is why I used two comments. :)

Reply:
The `interface` entity gets generated when the comment block is closed, or soemthing like this.
`interface` and `name` are two things and you need to close the block inbetween

Reply:
Thanks, then this is resolved :)